core/src/miner/mem_pool.rs

```diff
@@ -781,17 +770,18 @@ impl MemPool {
         origin: TxOrigin,
         client_account: &AccountDetails,
     ) -> Result<(), Error> {
-        if origin != TxOrigin::Local && tx.fee < self.minimal_fee {
+        let action_min_fee = self.minimum_fees.min_cost(&tx.action);
```

Comment:
Why `origin != TxOrigin::Local` condition is dropped?

Reply:
Because I think that's the correct behavior. Some services let users send their transactions via RPC directly to the nodes. In this case, I think the mem pool minimum fees should be applied to such transactions.

Reply:
Oh. The spec is changed. okay.

Reply:
I think that we should remain the original behavior. RPC should not be open to the internet. We didn't consider any safety in RPC. All the people who want to send a transaction should have its own CodeChain instance.

Reply:
How about discussing this with other team members?

Reply:
Ok then, I'll summary it and make a conversation. Until now, the mem pool's minimal fee is always set to zero.

Reply:
I changed the behavior